docs: What's New & Upgrade Guide 10.0

imatwawana opened the pull request:
This PR adds the What's New which describes new features and changes available in Grafana v10.0, the v10.0 Upgrade Guide, and the v10.0 Breaking Changes. This PR also updates the order of these pages in the docs.

imatwawana committed:
updated what's new index and added v10.0 what's new and upgrade guide

chri2547 committed:
Update _index.md

zsofiakm committed:
Added nested folders

zsofiakm committed:
Nested folders screenshot

josiahg committed:
Add draft of Correlations what's new content

josiahg committed:
Correlations minor edit

natellium committed:
add dashboards whats new

natellium commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,87 @@
+
+In subsequent releases, we’ll be refining and enhancing the user interface for Correlations, to provide a more streamlined user experience.
+
+## First panel creation flow
```
@imatwawana Could you check the content of `First panel creation flow`, `Public dashboards`, and `scenes`?

natellium commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,87 @@
+
+The new panel creation flow greatly simplifies the data source selection step. It provides a clear overview of your available data sources and a direct link to connecting a new one, the first time you add a panel to your dashboard.
+
+To try it out, you'll need to enable the `datasourceOnboarding` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
```
@ivanortegaalba Are we reusing the same feature toggle we had for the onboarding flow? Also, would be great to add a screenshot of the modal here, could you do that?

natellium commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,87 @@
+These are some of the improvements you will find in this version:
+
+<!--
+- TODO:Improvements
```
@evictorero @juanicabanas can we add a list of improvements we've done since the experimental release last year?

natellium commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,87 @@
+
+To try it out, you'll need to enable the `publicDashboards` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+To learn more, check out our public dashboards [documentation](https://grafana.com/docs/grafana/latest/dashboards/dashboard-public/?pg=blog&plcmt=body-txt).
```
@evictorero @juanicabanas we should review the docs, make sure they're up to date

natellium commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,87 @@
+
+## Scenes
+
+_This is an experiemental library_
```
@dprokop @torkelo can you review this? I also added a comment to add a link to docs and resources once we have them.

imatwawana replied:
@dprokop let me know that this shouldn't be nested under Dashboards so i'll leave it where it is.

ifrost committed:
What's new: Query multiple data sources in Explore

natellium committed:
Update docs/sources/whatsnew/whats-new-in-v10-0.md

Co-authored-by: Dominik Prokop <dominik.prokop@grafana.com>

vtorosyan committed:
AuthNZ: What's new in Grafana 10.0 (#66362)

* AuthNZ: What's new in Grafana 10.0

* Add upgrade guides

* Apply suggestions from code review

Co-authored-by: Ieva <ieva.vasiljeva@grafana.com>

* Apply suggestions from code review

Co-authored-by: Jo <joao.guerreiro@grafana.com>

* Apply suggestions from code review

Co-authored-by: Jo <joao.guerreiro@grafana.com>

---------

Co-authored-by: Ieva <ieva.vasiljeva@grafana.com>
Co-authored-by: Jo <joao.guerreiro@grafana.com>

thanos-karachalios commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,87 @@
+_Available in preview in all editions of Grafana._
+Concepts like data sources and dashboards panels are hard to grasp and it’s a struggle to go from Grafana’s “empty state” to a working dashboard that displays data.
+
+The new panel creation flow greatly simplifies the data source selection step. It provides a clear overview of your available data sources and a direct link to connecting a new one, the first time you add a panel to your dashboard.
```
@natellium I would also add the ability to upload a csv file as well(since can be seen a source of data)

joey-grafana committed:
Added span filters section

ivanortegaalba commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,141 @@
+The new panel creation flow greatly simplifies the data source selection step. It provides a clear overview of your available data sources and a direct link to connecting a new one, the first time you add a panel to your dashboard.
+
+To try it out, you'll need to enable the `datasourceOnboarding` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
```
Here is the presentation for the new Data source picker. 

```suggestion
## Introducing an Enhanced Data Source Selection Experience: Simplifying Querying for Users

Our latest advancements streamline the process of selecting the ideal data source in Grafana, prioritizing recent usage and providing labels and supplementary descriptions. The enhanced data source picker includes contextual information and the ability to upload CSV files directly, and an advanced mode with built-in sources and the option to add a new one.

These updates enhance the querying experience on our platform, making it easier for users to find the desired data source.

```

ivanortegaalba commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,141 @@
+## First panel creation flow
+
+_Available in preview in all editions of Grafana._
+Concepts like data sources and dashboards panels are hard to grasp and it’s a struggle to go from Grafana’s “empty state” to a working dashboard that displays data.
+
+The new panel creation flow greatly simplifies the data source selection step. It provides a clear overview of your available data sources and a direct link to connecting a new one, the first time you add a panel to your dashboard.
+
+To try it out, you'll need to enable the `datasourceOnboarding` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
```
```suggestion
Concepts like data sources and dashboards panels are hard to grasp and it’s a struggle to go from Grafana’s “empty state” to a working dashboard that displays data.

With the new flow, selecting a data source has been greatly simplified, providing a clear overview of available data sources and allowing users to quickly connect to a new one when needed. Additionally, the flow now enables quick uploading of CSV files and access to built-in data sources.

To try it out, you'll need to enable the `datasourceOnboarding` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
```

Adding @thanos-karachalios suggestion

ArturWierzbicki committed:
add info about dashboard previews removal

natellium committed:
Update docs/sources/whatsnew/whats-new-in-v10-0.md

Co-authored-by: Ivan Ortega Alba <ivanortegaalba@gmail.com>

oscarkilhed committed:
add datagrid and drag and drop

natellium committed:
combine new ds picker and panel onboarding content

svennergr committed:
Logs: Add LogContext to What's New v10 (#66575)

* add log context description

* change `eyeball button` to `eye icon`

brendamuir committed:
Adds Alerting entries for 10

aangelisc committed:
Add Azure Monitor entry for v10

aangelisc committed:
Merge branch 'docs/whats-new-v10' of github.com:grafana/grafana into docs/whats-new-v10

# Conflicts:
#	docs/sources/whatsnew/whats-new-in-v10-0.md

natellium committed:
add public dashboards features

juanicabanas commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,241 @@
+
+_Available in preview in Grafana Enterprise, Cloud Pro, and Cloud Advanced._
+
+The public dashboards insights provide valuable information about your dashboard usage. You can easily access and view important metrics such as the daily query count, the number of views in the last 30 days, and the number of errors in the last 30 days.
```
shouldn't we say "public dashboard usage"
```suggestion
The public dashboards insights provide valuable information about your public dashboard usage. You can easily access and view important metrics such as the daily query count, the number of views in the last 30 days, and the number of errors in the last 30 days.
```

juanicabanas commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,241 @@
+
+Our email sharing feature allows you to easily share your public dashboards with specific individuals. By adding their email addresses, they will receive a one-time valid link to access the dashboard for 30 days. This provides you with greater control over who can visualize your data.
+
+To try it out, you'll need to enable the `publicDashboardsEmailSharing` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
```
I think we should also mention the `publicDashboards` feature toggle, right?

juanicabanas reviewed (commented):
Let some suggestions

natellium committed:
small tweaks to public dashboards content

tskarhed committed:
Add Security: Trusted types section

nmarrs committed:
Add DataViz: initial placeholder sections with some content

brendamuir committed:
Adds alerting entry for templates

brendamuir committed:
Adds an and

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,275 @@
+## Authentication and authorization
+
+### Role-based access control is always enabled
+
```
@vtorosyan - Should there be a feature availability note (e.g., _Generally available in all editions of Grafana_)

imatwawana replied:
Ahhh okay, I understand. Will review your PR.

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,275 @@
+
+To try it out, you'll need to enable the `advancedDataSourcePicker` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+## Public dashboards
```
```suggestion
## Dashboards and visualizations

### Public dashboards
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,275 @@
+
+To learn more, check out our public dashboards [documentation](https://grafana.com/docs/grafana/latest/dashboards/dashboard-public/?pg=blog&plcmt=body-txt).
+
+## Public dashboards insights
```
```suggestion
### Public dashboards insights
```

imatwawana committed:
update headings

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,277 @@
+To try it out, you'll need to enable the `newTraceView` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+## Datagrid
+
```
@oscarkilhed - I want to nest your items (Datagrid and Drag and drop) under the Dashboards and visualizations heading that I've added. Any objections?

supilee replied:
@imatwawana - sounds good (@oscarkilhed is out)

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,277 @@
+
+### Public dashboards
+
+_Available in preview in all editions of Grafana._
```
```suggestion
_Available in preview in all editions of Grafana._

```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,277 @@
+### Public dashboards
+
+_Available in preview in all editions of Grafana._
+Public dashboards allow you to share your Grafana dashboard with anyone without them having to log into Grafana. This is useful when you want to expose your dashboard to the world.
```
Is it okay not to use "expose" here? The rest of the sentence seems pretty casual and "expose" seems like quite a technical term.
```suggestion
Public dashboards allow you to share your Grafana dashboard with anyone without requiring them to log in to Grafana. This is useful when you want to make your dashboard available to the world.
```

natellium replied:
Oops, I hadn't seen ehe comment. Looks good to me! Thanks @imatwawana :)

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,277 @@
+- The time picker and annotations can be toggled on or off from the configuration modal
+- You can see all the public dashboards from the Dashboards menu
+- Improved UI: new modal design, paused, and not found pages
+- Bug fixes! E.g collapsed rows, hidden queries, and zoom into panels
```
It's not clear whether we're describing the bug or the fix here. Can we clarify
```suggestion
- Bug fixes! E.g collapsed rows, hidden queries, and zoom into panels
```

natellium replied:
Fair enough. @juanicabanas @evictorero could we add some more details about what that means?

evictorero replied:
Panels inside collapsed rows, panels with hidden queries and zoom-in on panels were not working on public dashboards. Maybe we can change the "Bug fixes" to "Added support for..."?

imatwawana replied:
That works for me!

nmarrs commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,277 @@
+The list of requests is already growing and of course we will look into those. So stay tuned for more features coming in the near future that will give an even better experience using the the drag & drop datasource.
+
+## Canvas GA
+
```
Nope, that sounds good!

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,277 @@
+
+To try it out, you'll need to enable the `publicDashboards` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+To learn more, check out our public dashboards [documentation](https://grafana.com/docs/grafana/latest/dashboards/dashboard-public/?pg=blog&plcmt=body-txt).
```
We should be using relrefs rather than absolute links so that they always resolve no matter what version we point to
```suggestion
To learn more, check out our public dashboards [documentation]({{< relref "../../dashboards/dashboard-public" >}}).
```

imatwawana reviewed (commented):
@evictorero and @juanicabanas - I reviewed the public dashboards items. Please take a look.

juanicabanas commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,277 @@
+
+- The time picker and annotations can be toggled on or off from the configuration modal
+- You can see all the public dashboards from the Dashboards menu
+- Improved UI: new modal design, paused, and not found pages
```
It's paused and not found pages. Both (paused and not found) are two new different pages

imatwawana committed:
fix link format and update wording

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,277 @@
+Our email sharing feature allows you to easily share your public dashboards with specific individuals. By adding their email addresses, they will receive a one-time valid link to access the dashboard. This provides you with greater control over who can visualize your data.
+
+To try it out, you'll need to enable the `publicDashboardsEmailSharing` and `publicDashboards` feature toggles. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
```
```suggestion

### Datagrid

We introduced a new panel, which allows users to edit their data within a Grafana dashboard: The Datagrid
The Datagrid is a new beta panel that allows users to edit their data within a Grafana dashboard. Imagine having a spreadsheet-like view where you can fine-tune data pulled from a datasource or create your own dataset from scratch and use it within your dashboard to update your panels in real time.

In combination with other panels using the Datagrid as a datasource, you can enhance the experience even more.

By default, the Datagrid panel will not allow a user to edit data, but only show data like a standard visualization. To make full use of this new panel editing functionality you will need to enable the “enableDatagridEditing” feature toggle.
But for now the Datagrid Panel will support the following features in G10:

- Creating and deleting rows and columns
- Data and column header edit or delete
- Search functionality
- Column freezing
- Grid selection actions (copy/paste/delete)
- Draggable columns and rows
- Series select when pulling data from a datasource

To learn more about the Datagrid panel, please refer to the official documentation.

The list of features is already growing and of course we will look into those. So stay tuned for more features coming in the near future that will give an even better experience using the datagrid.

### Drag and Drop

Finally connecting your local data is made easier! Welcome drag & drop in Grafana!
The drag & drop functionality allows users to connect their csv, excel or numbers files by simply dragging and dropping them on a dashboard, or even into the editor directly.

Once dropped a table will be created by default containing the data that is living in the file itself.

By default, the drag & drop feature is not available out of the box. In order to utilize this functionality panel you will need to enable the `featuretogglename` feature toggle.

As of now drag & drop supports the following scenarios in G10:

- Drag & drop files on an empty dashboard
- Drag & drop files into the panel editor
- Replace files in the panel editor
- Default table panel creation

As of now the data is being stored on the dashboard json and has a 1MB size limit. Obviously it’s our mission to increase the limit without decreasing the performance in the near future.
To learn more about drag & drop, please refer to the official documentation.

The list of requests is already growing and of course we will look into those. So stay tuned for more features coming in the near future that will give an even better experience using the the drag & drop datasource.

```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,277 @@
+
+The list of requests is already growing and of course we will look into those. So stay tuned for more features coming in the near future that will give an even better experience using the the drag & drop datasource.
+
+## Canvas GA
+
+We are promoting the canvas panel out of beta and into general availability. Over the past several months we have introduced several enhancements to the panel such as the ability to draw connections between elements, the ability to add data links, and a myriad of UX improvements. We are excited to include canvas as a first class citizen in Grafana’s core panel library. To learn more about the Can ...
+
+## New Trend Panel
+
+_Available in **beta** in all editions of Grafana_
+
+We are excited to introduce a new panel that allows you to display trends where the x axis is not time. This new beta panel addresses gaps that were not solved by either the Time series or XY Chart panels.
+
+## Time series time region support
+
+We have implemented support for adding time regions to the time series panel. Time regions provide a more contextualized experience, enabling you to highlight certain days of the week, such as Monday-Friday to display work weeks right alongside your data. Time regions are a useful way to highlight specific parts of a day like night, work hours, or whatever you define for each day. They allow the  ...
+
+## Annotation filtering
+
+We’ve improved the way you can configure annotations by adding the possibility to apply annotations to all panels, selected panels or to select the panels by exclusion.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-0-annotation-filtering.png" max-width="750px" caption="Annotation filtering" >}}
+
```
```suggestion
```

imatwawana committed:
nesting topics under dashboards and visualizations, updated headers

brendamuir committed:
Adds oncall contact point entry

brendamuir committed:
Copyedits for oncall alerting entry

gtk-grafana committed:
add incremental querying blurb to whats-new

zsofiakm committed:
Renaming nested folders to subfolders

vtorosyan committed:
Update what's new to indicate availability of features and improve messaging (#66755)

adela-almasan committed:
Added image for time regions

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,292 @@
+
+<!-- Template below
+## Feature
+[Generally available | Available in experimental/beta] in Grafana [Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced]
```
```suggestion
[Generally available | Experimental | In private/public preview] in Grafana [Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced]
```

imatwawana committed:
update wording for feature release stage

aocenas committed:
Add section about Phlare data source rename

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,296 @@
+
+To try it out, you'll need to enable the `advancedDataSourcePicker` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+## Dashboards and visualizations
+
```
```suggestion
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,296 @@
+To try out Correlations, you'll need to enable the `correlations` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+In subsequent releases, we’ll be refining and enhancing the user interface for Correlations, to provide a more streamlined user experience.
+
```
```suggestion

## Dashboards and visualizations

```

imatwawana committed:
restructuring and styling fixes

natellium committed:
add datasource picker recording

natellium commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -95,9 +95,7 @@ Concepts like data sources and dashboards panels are hard to grasp and it’s a
-<!--
-- TODO:Add screenshots
--->
+{{< video-embed src="/media/docs/grafana/screen-recording-ds-picker-whats-new-10.mp4" max-width="750px" caption="Datasource picker flow" >}}
```
@imatwawana datasource picker recording 👍

ivanortegaalba replied:
@natellium I'll upload a new video to the same URL, replacing the previous version, so no changes here are needed :)

svennergr committed:
Log Context: Improved "Whats New" text (#67285)

* improve "Log context" whats new

* change screenshot

* Update docs/sources/whatsnew/whats-new-in-v10-0.md

Co-authored-by: Matias Chomicki <matyax@gmail.com>

* fix spelling

* Update docs/sources/whatsnew/whats-new-in-v10-0.md

Co-authored-by: J Stickler <julie.stickler@grafana.com>

* Update docs/sources/whatsnew/whats-new-in-v10-0.md

Co-authored-by: J Stickler <julie.stickler@grafana.com>

* Update docs/sources/whatsnew/whats-new-in-v10-0.md

Co-authored-by: J Stickler <julie.stickler@grafana.com>

* Update docs/sources/whatsnew/whats-new-in-v10-0.md

Co-authored-by: J Stickler <julie.stickler@grafana.com>

* change image

---------

Co-authored-by: Matias Chomicki <matyax@gmail.com>
Co-authored-by: J Stickler <julie.stickler@grafana.com>

imatwawana committed:
wording and style edits

tskarhed committed:
Update how to use Trusted Types

tolzhabayev committed:
Adding opentracing deprecation notes

tolzhabayev committed:
Adding opentracing deprecation notes

imatwawana committed:
added breaking changes with raw markdown

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,306 @@
+  - '10.0'
+  - release notes
+title: What's new in Grafana v10.0
+weight: -33
```
```suggestion
weight: -37
```

imatwawana committed:
updated page weight

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,306 @@
+
+_Available in preview in Grafana Enterprise, Cloud Pro, and Cloud Advanced._
+
+Our email sharing feature allows you to easily share your public dashboards with specific individuals. When you add their email addresses, they will receive a one-time link to access the dashboard. This provides you with greater control over who can visualize your data.
```
@juanicabanas @natellium 

```suggestion
Our email sharing feature allows you to easily share your public dashboards with specific individuals. When you add their email addresses, they will receive a one-time link to access the dashboard. This provides you with greater control over who can visualize your data.

We've also added a **Public dashboard users** tab in **Administration > Users** where you can view a list of users who have accessed your public dashboards by way of a email sharing.
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,306 @@
+With this update, we've made the following improvements:
+
+- The time picker and annotations can be toggled on or off from the configuration modal
+- You can see all the public dashboards from the Dashboards menu
```
```suggestion
- You can see a list of all your public dashboards at **Dashboards > Public dashboards**
```

imatwawana committed:
state location of public dashboards list more explicitly

imatwawana committed:
added email sharing user admin info

imatwawana committed:
clarified what was fixed

brendamuir committed:
Deletes oncall entry

imatwawana committed:
added heading markdown

imatwawana committed:
replaced absolute links with relrefs

imatwawana committed:
fixed relrefs

nmarrs committed:
Update canvas GA section to include connection properties

nmarrs committed:
canvas: slight grammar update

imatwawana committed:
updated template and name of contributor field for each item

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,359 @@
+
+## Scenes
+
+<!-- Name of contributor -->
```
```suggestion
<!-- Dominik Prokop -->
```

imatwawana committed:
adding contributor name

imatwawana committed:
added contributor names to what's new

imatwawana committed:
added contributor names to what's new and upgrade guide

imatwawana committed:
fixed typo

timlevett commented on docs/sources/breaking-changes/breaking-changes-v10-0.md:
```diff
@@ -0,0 +1,265 @@
+
+#### Change in Grafana v10
+
+Angular will be disabled by default for all new Grafana Cloud stacks.
```
Also @tolzhabayev will we block customers from installing angular plugins from new stacks that have them turned off? Could improve the UX if we did.

sympatheticmoose replied:
Updated the text for the first comment. 

> will we block customers from installing angular plugins from new stacks

This isn't possible _yet_ but is something we're working on.

timlevett commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,359 @@
+You can now configure SAML using our new user interface, making the process easier and more convenient than ever before.
+With the new user interface, you can now configure SAML without needing to restart Grafana and you can control access to the configuration UI by using [role-based access control (RBAC)]({{< relref "../administration/roles-and-permissions/access-control/" >}}). which makes the process much faster and more efficient.
+
+The SAML UI is available in Grafana Enterprise and Grafana Cloud Pro and Advanced. It is intuitive and user-friendly, with clear instructions and helpful prompts to guide you through the process.
```
@vtorosyan what do you think of telling users how to navigate to the UI in this? I tried to read this to learn where to go. Without this guidance, i went to top nav and typed in "SAML" and got no results, then i went to administration, and there is not an option for SAML... I then went to the docs, where after the "pre" section I'm presented with the path. Also noteworthy that you have that path setup as "Administration"->"Authentication"-> "SAML Configuration" but the actual tile is just "SAML".

timlevett commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,359 @@
+
+To try it out, you'll need to enable the `publicDashboards` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+### Email sharing for public dashboards
```
@thanos-karachalios  We may want to rephrase this to 
>Email Sharing for Dashboards

The title here could be confused that the dashboard is public, but its also shared via email. Removing public from here could clear this up. Either that or you could mention below that the dashboard is only visible by the people you share it with.

timlevett replied:
cc @evictorero

timlevett commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,359 @@
+
+<!-- Nathan Marrs -->
+
+_Available in **beta** in all editions of Grafana_
```
@nmarrs we should use the term preview, not beta. cc @imatwawana See https://docs.google.com/spreadsheets/d/19GMKK09PAFZL6G5V54RpPZrnnTLGOuREVZVhk6q6Zjk/edit#gid=776746502 for details.

timlevett commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,359 @@
+
+We have implemented support for adding time regions to the time series panel. Time regions provide a more contextualized experience, enabling you to highlight certain days of the week, such as Monday-Friday to display work weeks right alongside your data. Time regions are a useful way to highlight specific parts of a day like night, work hours, or whatever you define for each day. They allow the  ...
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-0-time-regions.png" max-width="750px" caption="Time regions" >}}
```
@nmarrs We could very shortly say where to find this in the configuration of a time series panel, or link to a page to learn more how to set this up for a panel. I took a while to try to find it in our 10.x test instance and I couldn't find it for a timeseries panel.

nmarrs replied:
I agree on this. Time regions are set up through dashboard annotations instead of a specific panel editor option

nmarrs replied:
done 👍 [commit here](https://github.com/grafana/grafana/pull/64927/commits/bdbd773d78414759c4540e230f29ce589e1f31a7)

kylebrandt committed:
Docs: What is new 10 - add Data plane (#68125)

oscarkilhed committed:
Remove drag & drop to dashboard

mjseaman committed:
Add PDC to 10.0 what's new

mjseaman committed:
reorder sections, edit copy

mjseaman committed:
update order, format and copy

mjseaman committed:
grammar fix

add "and"

imatwawana commented on docs/sources/breaking-changes/_index.md:
```diff
@@ -0,0 +1,23 @@
+
+# Upgrade Grafana
+
+<!-- TO DO add intro text -->
```
In some cases, major releases that introduce many new features, also introduce breaking changes. These changes are described, along with information about what to do, in the breaking changes pages specific to each release.

For our purposes, a breaking change is any change that requires users or operators to do something. This includes:

- Changes in one part of the system that could cause other components to fail
- Deprecations or removal of a feature
- Changes to an API that could break automation
- Changes that affect some plugins or functions of Grafana
- Migrations that can’t be rolled back

{{% admonition type="note" %}}

To learn what's available in a Grafana release, refer to [What's new ](link) page for each version. For the steps we recommend when you upgrade, check out the [Upgrade guide](link) for each version.

{{% /admonition %}}

Refer to any of the following breaking changes guides:

{{< section menuTitle="true">}}

imatwawana commented on docs/sources/breaking-changes/_index.md:
```diff
@@ -0,0 +1,23 @@
+
+# Upgrade Grafana
+
+<!-- TO DO add intro text -->
+
+To learn what's available in a Grafana release, refer to [What's New ]({{< relref "../whatsnew/" >}}).
+
+Refer to any of the following upgrade guides:
+
+{{< section menuTitle="true">}}
```
```suggestion
In some cases, major releases that introduce many new features, also introduce breaking changes. These changes are described, along with information about what to do, in the breaking changes pages specific to each release.

For our purposes, a breaking change is any change that requires users or operators to do something. This includes:

- Changes in one part of the system that could cause other components to fail
- Deprecations or removal of a feature
- Changes to an API that could break automation
- Changes that affect some plugins or functions of Grafana
- Migrations that can’t be rolled back

{{% admonition type="note" %}}

To learn what's available in a Grafana release, refer to the [What's new ]({{< relref "../whatsnew/" >}})) page for each version. For the steps we recommend when you upgrade, check out the [Upgrade guide]({{< relref "../upgrade-guide/" >}}) for each version.

{{% /admonition %}}

Refer to any of the following breaking changes guides:

{{< section menuTitle="true">}}
```

imatwawana commented on docs/sources/breaking-changes/breaking-changes-v10-0.md:
```diff
@@ -0,0 +1,265 @@
+
+# Breaking changes in Grafana v10.0
+
+<!-- TO DO add intro text -->
```
```suggestion
Following are breaking changes that you should be aware of when upgrading to Grafana v10.0. 

For our purposes, a breaking change is any change that requires users or operators to do something. This includes:

- Changes in one part of the system that could cause other components to fail
- Deprecations or removal of a feature
- Changes to an API that could break automation
- Changes that affect some plugins or functions of Grafana
- Migrations that can’t be rolled back

For each change, the provided information:

- Helps you determine if you’re affected
- Describes the change or relevant background information
- Guides you in how to mitigate for the change or migrate
- Provides more learning resources

For release highlights and deprecations, refer to our [v10.0 What’s new]({{< relref "../whatsnew/whats-new-in-v10.0/index.md" >}}). For the specific steps we recommend when you upgrade to v10.0, check out our [Upgrade guide]({{< relref "../upgrade-guide/upgrade-v10.0/index.md" >}}).

```

imatwawana committed:
add intro text to breaking changes pages

evictorero commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,375 @@
+
+<!-- Juani Cabanas & Ezequiel Victorero & Natalia Bernarte -->
+
+_Available in preview in Grafana Enterprise, Cloud Pro, and Cloud Advanced._
+
+Our email sharing feature allows you to easily share your public dashboards with specific individuals. When you add their email addresses, they will receive a one-time link to access the dashboard. This provides you with greater control over who can view your public dashboards.
+
+We've also added a **Public dashboard users** tab in **Administration > Users** where you can view a list of users who have accessed your public dashboards by way of a email sharing.
+
+To try it out, you'll need to enable the `publicDashboardsEmailSharing` and `publicDashboards` feature toggles. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
```
```suggestion
_Available in preview in Cloud Pro, and Cloud Advanced._

Our email sharing feature allows you to easily share your public dashboards and make them visible only with specific individuals. When you add their email addresses, they will receive a one-time link to access the dashboard. This provides you with greater control over who can view your public dashboards.

We've also added a **Public dashboard users** tab in **Administration > Users** where you can view a list of users who have accessed your public dashboards by way of a email sharing.

To try it out, please contact customer support.

This feature will have a cost by active users after being promoted into general availability.
```

evictorero replied:
@imatwawana please help me improve these changes:
 * Changed versions, now only available on Cloud
 * Removed any mentions of feature flags
 * Added a comment to state that only users that received an invitation will have access to the public dashboard as @timlevett suggested.
 * Explained this feature will have a cost
 
cc: @thanos-karachalios @natellium

imatwawana replied:
I think this is fine; I've made one small wording tweak and added "private" to the availability note, to denote that this feature is only available to invited customers.

evictorero replied:
I think this fits better with the concept of `cloud public preview` from our new [Release life cycle](https://docs.google.com/document/d/11SD-DSBtHDyXOzPVlq82cwy3XeeBhqClejUorUi86Hw/edit#) because we are announcing it publicly and it will available to anyone who requests access.

imatwawana replied:
Ah okay, you're right. I misunderstood.

Elfo404 committed:
fix Explore mixed DS what's new doc (#68303)

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

oscarkilhed committed:
Remove datagrid from what's new, drag and drop to experimental

evictorero committed:
Update docs/sources/whatsnew/whats-new-in-v10-0.md

ivanortegaalba commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,356 @@
+
+<!-- TODO: Add docs link above -->
+
+### Select your data source more easily
+
+<!-- Ivan Ortega & Natalia Bernarte -->
+
+_Available in preview in all editions of Grafana._
+
+Concepts like data sources and dashboards panels are hard to grasp and it’s a struggle to go from Grafana’s “empty state” to a working dashboard that displays data. Our latest advancements streamline the process of selecting the ideal data source in Grafana, prioritizing recent usage and providing labels and supplementary descriptions.
```
@imatwawana since we will enable the toggle by default, it will be generally available. Just removing this sentence is enough for the What's new, isn't it? Thanks!

```suggestion
### Select your data source more easily

<!-- Ivan Ortega & Natalia Bernarte -->

Concepts like data sources and dashboards panels are hard to grasp and it’s a struggle to go from Grafana’s “empty state” to a working dashboard that displays data. Our latest advancements streamline the process of selecting the ideal data source in Grafana, prioritizing recent usage and providing labels and supplementary descriptions.
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,356 @@
+
+{{< video-embed src="/media/docs/grafana/screen-recording-ds-picker-whats-new-10.mp4" max-width="750px" caption="Datasource picker flow" >}}
+
+To try it out, you'll need to enable the `advancedDataSourcePicker` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
```
this should go as well @ivanortegaalba 

```suggestion
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,356 @@
+
+<!-- Ivan Ortega & Natalia Bernarte -->
+
+_Available in preview in all editions of Grafana._
```
@ivanortegaalba - we'll want to change the note to this

```suggestion
_Generally available in all editions of Grafana._
```

imatwawana reviewed (commented):
@ivanortegaalba - i've made a couple edits

imatwawana committed:
updates to data source picker availability

nmarrs committed:
Update what's new for canvas / trend panel (finalized)

sympatheticmoose commented on docs/sources/breaking-changes/breaking-changes-v10-0.md:
```diff
@@ -0,0 +1,283 @@
+
+#### You are affected if
+
+You create a new Grafana Cloud stack and intend to use any panel or data source plugins written using the Angular.js JavaScript framework. See the list of Angular plugins here.
```
The list is still in review so we may need to update this page when it is live - https://github.com/grafana/grafana/pull/68096

sympatheticmoose commented on docs/sources/breaking-changes/breaking-changes-v10-0.md:
```diff
@@ -0,0 +1,283 @@
+
+If you are using Angular plugins in Grafana, try an alternative panel or data source that does not use Angular. Here is a list of the most-used Angular plugins, and their alternatives where they exist.
+
+If you are a developer maintaining a plugin that uses Angular, we recommend you refactor or rewrite its frontend using React.js instead. The team at Grafana is ready to help and provide tools and guidance; you can reach us here.
```
```suggestion
If you are a developer maintaining a plugin that uses Angular, we recommend you refactor or rewrite its frontend using React.js instead. The team at Grafana is ready to help and provide tools and guidance; you can reach us in [Slack](https://grafana.slack.com/archives/C3HJV5PNE) or our [Forum](https://community.grafana.com/c/plugin-development/30).
```

natellium committed:
Update ds picker recording

natellium commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -120,7 +120,7 @@ Concepts like data sources and dashboards panels are hard to grasp and it’s a
 With this flow, selecting a data source has been greatly simplified, providing a clear overview of available data sources and allowing you to quickly connect to a new one when needed. You can also quickly upload a CSV file.
 
-{{< video-embed src="/media/docs/grafana/screen-recording-ds-picker-whats-new-10.mp4" max-width="750px" caption="Datasource picker flow" >}}
+{{< video-embed src="/media/docs/grafana/screen-recording-ds-picker-whats-new-10-final.mp4" max-width="750px" caption="Datasource picker flow" >}}
```
@ivanortegaalba Turns out we can't delete or change files, so I had to add a new one.

papagian committed:
Add deprecation notice for changing folder UID

papagian commented:
> Which file should the deprecations notifications (not breaking anything) go into?
> 
> e.g. "Starting with , is deprecated. It will be removed in a future release."

I have added a section deprecations in [docs/sources/whatsnew/whats-new-in-v10-0.md](https://github.com/grafana/grafana/pull/64927/files#diff-8320b579469dd378e3ef71b0ed79650df605939aadbe0d6097e4402ef84b4530)
Not sure if it's how we usually do though.

sympatheticmoose committed:
docs: update with angular plugin list

sympatheticmoose commented:
Updated for links to new list of angular plugins, may need to rebase for the link checks to succeed?

adela-almasan committed:
Update with documentations links for annotation filtering and time regions

nmarrs commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,364 @@
+
+<!-- Andrej Ocenus -->
+
+As a part of ongoing unification of Phlare and Pyroscope project we are renaming the Phlare data source to Grafana Pyroscope data source. This data source will support both Phlare and Pyroscope backends. Existing instances of the data source should not be affected. When creating new instance of the data source, backend type will be autodetected on the config page, or you can select it manually.
```
```suggestion
We are renaming the Phlare data source to the Grafana Pyroscope data source as part of the ongoing unification of the Phlare and Pyroscope projects. This data source will support both Phlare and Pyroscope backends. Existing instances of the data source should not be affected. When creating a new instance of the data source, the backend type will be autodetected on the config page, or you can select it manually.
```

nmarrs commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,364 @@
+
+In order to query these data sources from Grafana Cloud, currently you have to open your private network to a range of IP addresses - this is a non-starter for a lot of IT Security teams. So the challenge is, how do you connect to your private data from Grafana Cloud, without exposing your network?
+
+The answer is Private Data Source Connect (PDC), available now in Public Preview in Grafana Cloud Pro and Advanced. PDC uses SOCKS over SSH to establish a secure connection between a lightweight PDC agent you deploy on your network and your Grafana Cloud stack. PDC keeps the network connection totally under your control. It’s easy to set up and manage, uses industry-standard security protocols, a ...
```
Do we need to update the link to docs to be relative vs public url?

nmarrs reviewed (commented):
was reviewing DataViz squad contributions but decided to give everything a read over and found a few potential grammar fixes. Amazing work from the entire Grafana team!

nmarrs committed:
Update annotation filtering section in docs/sources/whatsnew/whats-new-in-v10-0.md

knylander-grafana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,364 @@
+- Duration,
+- Tags (which include tags, process tags, and log fields).
+
+To try it out, you'll need to enable the `newTraceView` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
```
```suggestion
To try it out, you'll need to enable the `newTraceViewHeader` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
```

mdvictor committed:
add datagrid as experimental

mdvictor committed:
modify datagrid doc accordingly

joey-grafana committed:
Update docs/sources/whatsnew/whats-new-in-v10-0.md

Co-authored-by: Kim Nylander <104772500+knylander-grafana@users.noreply.github.com>

joey-grafana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,387 @@
+
+<!-- Timur Olzhabayev -->
+
+_Available in preview in all editions of Grafana._
```
For that phrase, I'm not entirely sure if it's correct. The [wording here](https://grafana.com/docs/release-life-cycle/) would suggest it is and it isn't, so not sure should this be updated. It's only going to be toggled on by default for hosted Grafana customers, using [release waves](https://github.com/grafana/deployment_tools/tree/master/ksonnet/environments/hosted-grafana/waves/feature-toggles#instructions) i.e. enabling by default in the canary and prod waves. This is a relatively new feature so it's not GA just yet but we would like to make it easier for customers to give us feedback.

knylander-grafana replied:
@joey-grafana  I went through some of the other docs and found this phrasing for preview features. What do you think? Is this a public preview or a private preview? I'm guessing public because the feature will be available for hosted Grafana. I updated the wording slightly for public preview. What do you think? 

The paragraph where you mention the feature flag also talks about availability for the feature. Does the combination of that paragraph and the suggested wording below cover things?
```suggestion
_Span filtering for traces is currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available._
```

joey-grafana replied:
Hi Kim, thank you. The wording here makes sense although I would imagine we would not introduce any breaking changes - more like the UI design may change. It sounds more like a public preview than a private one. I would say that you can make the call on the final wording we use together with whomever is in charge of reviewing this PR. This new feature will be available in Grafana Cloud only for the time being.

imatwawana committed:
changed order of sections

imatwawana committed:
updated text about breaking changes in whats new and upgrade guide

imatwawana committed:
fixed links

imatwawana committed:
copy edits

brendamuir committed:
adds alerting screenshot

brendamuir committed:
deletes sentence

brendamuir committed:
adds state history pic

imatwawana committed:
resolving outstanding suggestions

Co-authored-by: David Harris <david.harris@grafana.com>
Co-authored-by: Nathan Marrs  <nathanielmarrs@gmail.com>
Co-authored-by: Kim Nylander <104772500+knylander-grafana@users.noreply.github.com>

imatwawana committed:
general copy edits

imatwawana committed:
removed TO DOs from whats new, copy edited upgrade guide, some copy edits of breaking changes

imatwawana committed:
Merge branch 'main' into docs/whats-new-v10

imatwawana commented on docs/sources/upgrade-guide/_index.md:
```diff
@@ -20,6 +20,8 @@ Because Grafana upgrades are backward compatible, the upgrade process is straigh
 
 To learn what's available in a Grafana release, refer to [What's New ]({{< relref "../whatsnew" >}}).
 
+To learn about breaking changes in a Grafana release, refer to [Breaking changes]({{< relref "../breaking-changes/" >}}).
```
```suggestion
To learn about breaking changes in a Grafana release, refer to [Breaking changes]({{< relref "../breaking-changes" >}}).
```

imatwawana commented on docs/sources/upgrade-guide/upgrade-v10.0/index.md:
```diff
@@ -0,0 +1,94 @@
+<!-- Vardan Torosyan -->
+
+Usernames and email addresses are now treated as case-insensitive in Grafana. If you're not using MySQL as a database, potential user identity conflicts may arise when users try to log in.
+We recommend you resolve any potential conflicts in advance by using the [Grafana CLI tool for managing user conflicts](https://grafana.com/blog/2022/12/12/guide-to-using-the-new-grafana-cli-user-identity-conflict-tool-in-grafana-9.3/).
```
```suggestion
We recommend you resolve any potential conflicts in advance by using the [Grafana CLI tool for managing user conflicts](/blog/2022/12/12/guide-to-using-the-new-grafana-cli-user-identity-conflict-tool-in-grafana-9.3/).
```

imatwawana committed:
fix doc validator errors

vtorosyan commented:
@mjseaman @imatwawana pointed out that [RBAC change which makes it always on](https://github.com/grafana/grafana/pull/66755/files#diff-8320b579469dd378e3ef71b0ed79650df605939aadbe0d6097e4402ef84b4530R25) is missing from the What's new content now. It seems that it was removed in [this commit](https://github.com/grafana/grafana/commit/4b7810255960cdb3382df4f619afa0d4ce6887cd). 

Note that I can still see the change described in [Breaking changes](https://github.com/grafana/grafana/pull/64927/files#diff-158eeacea61be69505a37c957f0b6397d0706a415e0429e7da01eca4e6a63573R124) and in [the upgrade guide](https://github.com/grafana/grafana/pull/64927/files#diff-a437cfb0f87d2f40032143832fa624c6ad8140d3922b0e34cf212ac4f97178e0R19). 

Was the change intentional? Should we bring it back or leave things as they are?

mjseaman commented:
> @mjseaman @imatwawana pointed out that [RBAC change which makes it always on](https://github.com/grafana/grafana/pull/66755/files#diff-8320b579469dd378e3ef71b0ed79650df605939aadbe0d6097e4402ef84b4530R25) is missing from the What's new content now. It seems that it was removed in [this commit](https://github.com/grafana/grafana/commit/4b7810255960cdb3382df4f619afa0d4ce6887cd).
> 
> Note that I can still see the change described in [Breaking changes](https://github.com/grafana/grafana/pull/64927/files#diff-158eeacea61be69505a37c957f0b6397d0706a415e0429e7da01eca4e6a63573R124) and in [the upgrade guide](https://github.com/grafana/grafana/pull/64927/files#diff-a437cfb0f87d2f40032143832fa624c6ad8140d3922b0e34cf212ac4f97178e0R19).
> 
> Was the change intentional? Should we bring it back or leave things as they are?

Hey @vtorosyan, I thought this change was appropriate for the breaking changes/upgrade guide but not relevant for What's New, which is more focused on user-visible improvements. For the vast majority of users this will be an invisible change, and for the few customers it affects, it's something to consider during upgrade rather than a product improvement to try out.

If you disagree just lmk.

imatwawana committed:
added availability information, updated preview notes, replaced screenshot

imatwawana committed:
Merge branch 'main' into docs/whats-new-v10

imatwawana committed:
copy edit, added links, added breaking changes content

mjseaman commented on docs/sources/shared/upgrade/upgrade-common-tasks.md:
```diff
@@ -8,7 +8,9 @@ Because Grafana upgrades are backward compatible, the upgrade process is straigh
 {{% admonition type="note" %}}
-There might be minor breaking changes in some releases. We outline these changes in the [What's New ]({{< relref "../../whatsnew/" >}}) document for each release.
+
+> **Note:** There might be minor breaking changes in some releases. We outline these changes in the [What's New ]({{< relref "../../whatsnew/" >}}) document for most releases or the [Breaking changes]({{< relref "../../breaking-changes/" >}}), in some cases.
```
```suggestion
> **Note:** There might be breaking changes in some releases. We outline these changes in the [What's New ]({{< relref "../../whatsnew/" >}}) document for most releases or a separate [Breaking changes]({{< relref "../../breaking-changes/" >}}) document for releases with many breaking changes.
```

imatwawana replied:
Good call. I'll make this change on my local and push it since I've got some edits going.

vtorosyan commented:
@imatwawana please see @mjseaman answer [here](https://github.com/grafana/grafana/pull/64927#issuecomment-1566719305) (and thank you for the clarification @mjseaman!). 

I generally agree with a point that the change is not appropriate to include in what's new and as soon as we cover it under breaking changes, duplicating that information would not help much. That said, please feel free to do changes as you find it appropriate or else advice how to proceed and happy to do the changes myself.

imatwawana commented:
> @imatwawana please see @mjseaman answer [here](https://github.com/grafana/grafana/pull/64927#issuecomment-1566719305) (and thank you for the clarification @mjseaman!).
> 
> I generally agree with a point that the change is not appropriate to include in what's new and as soon as we cover it under breaking changes, duplicating that information would not help much. That said, please feel free to do changes as you find it appropriate or else advice how to proceed and happy to do the changes myself.

This works for me! We'll leave everything as it currently is.

qjohnson89 commented on docs/sources/_index.md:
```diff
@@ -70,8 +70,9 @@ title: Grafana documentation
+    <a href="{{< relref "whatsnew/whats-new-in-v10-0/" >}}" class="nav-cards__item nav-cards__item--guide">
+        <h4>What's new in v10</h4>
         <p>Explore the features and enhancements in the latest release.</p>
     </a>
```
Welcome to Grafana 10! We are excited to introduce a series of powerful enhancements in areas including user experience, security, and plugin functionality. Read on to understand more about these significant changes in search and navigation, dashboards and visualizations, and authentication and security. For a more comprehensive dive into all the transformations in this release, please refer to the complete Grafana 10 documentation.

In this release, we've strongly emphasized creating a more intuitive user experience and delivering robust security measures. From bolstering our authentication protocols to debuting a host of new features like Grafana Correlations, Subfolders, Public Dashboards, Scenes, and new visualizations, Grafana 10 is designed to empower teams with the best tools for data visualization and monitoring. Stay tuned to discover how Grafana 10 can streamline your workflows, improve incident resolution, and enhance data security.

imatwawana committed:
general copy edit of breaking changes; small update to common tasks

imatwawana committed:
replaced notes with admonitions

imatwawana committed:
link fixes

imatwawana committed:
reverted blog links to full addresses

imatwawana committed:
fixed heading inconsisency

imatwawana committed:
fix doc validator error

torkelo commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,411 @@
+
+_Available in public preview in all editions of Grafana._
+
+You can now create subfolders in Grafana to help you better organize your dashboards and alerts. This new feature allows you to create, read, update, and delete subfolders, making it easier to sort resources by business units, departments, and teams.
+
+You can also set up permissions using Role-Based Access Control (RBAC). Folder permissions cascade, being inherited from the parent folder, which simplifies access management.
+
+The ability to add subfolders is currently in preview. As such, it's recommended to enable it only on test or development instances, rather than in production environments.
+
+To get started creating subfolders, enable the `nestedFolders` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+In subsequent releases, we’ll be refining and enhancing the user interface for managing dashboards and folders, to provide a more streamlined user experience.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10.0-nested-folders-2.png" max-width="750px" caption="Subfolders in Grafana" >}}
+
```
@zsofiakm 
I am not sure we can include this, unless we plan to backport https://github.com/grafana/grafana/pull/68124 to G10

zsofiakm replied:
Updated the description so that it calls out that the feature is missing.

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,411 @@
+
+<!-- Timur Olzhabayev -->
+
+We've started the work to migrate to OpenTelemetry in Grafana version 8.4; now we're removing OpenTracing and, for those who still have it configured, replacing it under the hood with OpenTelemetry. These changes are backwards compatible, so you don't need to change anything the feature will continue working as it did before.
```
```suggestion

_Generally available in all editions of Grafana._

We've started the work to migrate to OpenTelemetry in Grafana version 8.4; now we're removing OpenTracing and, for those who still have it configured, replacing it under the hood with OpenTelemetry. These changes are backwards compatible, so you don't need to change anything the feature will continue working as it did before.
```

imatwawana committed:
added availability note

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,413 @@
+### Data plane
+
+<!-- Kyle Brandt -->
+
```
```suggestion

_Generally available in all editions of Grafana._

```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,413 @@
+
+<!-- Nathan Marrs -->
+
+We're promoting the canvas panel out of beta and into general availability. Over the past several months we've introduced multiple enhancements to the panel such as the ability to draw connections between elements, the ability to set the color and size of connections based on data, and the ability to add data links. We're excited to include Canvas as a first class citizen in Grafana’s core panel  ...
```
```suggestion
We're promoting the canvas panel out of public preview and into general availability. Over the past several months we've introduced multiple enhancements to the panel such as the ability to draw connections between elements, the ability to set the color and size of connections based on data, and the ability to add data links. We're excited to include Canvas as a first class citizen in Grafana’s core panel library. To learn more about the panel, refer to our [Canvas documentation]({{< relref "../panels-visualizations/visualizations/canvas" >}}). Also, check out our [latest blog post about canvas](https://grafana.com/blog/2023/05/11/use-canvas-panels-to-customize-visualizations-in-grafana/).
```

imatwawana committed:
added missing availability notes

imatwawana committed:
replaced screenshot and fixed wording

imatwawana committed:
replaced screenshot and fixed wording

zsofiakm committed:
Updated subfolders description

To call out missing features and stress that it can only be used in dev/test environments

imatwawana commented on docs/sources/whatsnew/whats-new-in-v10-0.md:
```diff
@@ -0,0 +1,413 @@
+- [Go library for reading and writing dataplane data](https://github.com/grafana/dataplane/tree/main/sdata)
+
+## Alerting
+
```
```suggestion

_All Alerting features are generally available in all editions of Grafana._

```

imatwawana committed:
added availability note

chri2547 reviewed (approved):
LGTM! Thanks for all the great work on this document!
